framework/core/js/src/admin/AdminApplication.tsx

```diff
@@ -32,6 +32,7 @@ export type Extension = {
   extra: {
     'flarum-extension': {
       title: string;
+      'database-support': undefined | string[];
```

Comment:
Should this be a union type instead of a generic string?

Reply:
We can't really ensure people will actually use the exact strings mysql, pgsql, sqlite

It's very likely someone will write something different, cause that's just how user input is.

Reply:
Fair enough, we could try to validate, but it's such a low priority, I'm not sure it's worth it.